src/yadism/coefficient_functions/asy/partonic_channel.py

```diff
@@ -155,3 +155,24 @@ def decorator(self, f):
         if self.is_below_pair_threshold(self.ESF.x):
             return lambda: pc.RSL()
         return f
+
+    def is_below_pair_threshold(self, z):
```

Comment:
since they are scoped back under NC by the parent class name you could (or not) remove the `_pair` again :upside_down_face:

Reply:
Yes I could, but I think this name is also fine